bindings/python/test.py

```diff
@@ -256,6 +256,14 @@ def test_sha1hash(self):
 		s = lt.sha1_hash(binascii.unhexlify(h))
 		self.assertEqual(h, str(s))
 
+class test_magnet_link(unittest.TestCase):
+
+	def test_parse_magnet_uri(self):
+		ses = lt.session({})
+		magnet = 'magnet:?xt=urn:btih:C6EIF4CCYDBTIJVG3APAGM7M4NDONCTI'
+		p = lt.parse_magnet_uri(magnet)
+		p['save_path'] = '.'
+		h = ses.add_torrent(p)
```

Comment:
Should have an assert that will also show up the second issue; incorrect hash calculated:

```
self.assertEqual(str(h.info_hash()), '178882f042c0c33426a6d81e0333ece346e68a68')
```